Gemfile.lock

```diff
@@ -9,8 +9,10 @@ GEM
       eventmachine (>= 0.12.9)
       http_parser.rb (~> 0)
     eventmachine (1.2.7)
-    ffi (1.16.3)
-    ffi (1.16.3-x64-mingw-ucrt)
+    ffi (1.17.0-arm64-darwin)
+    ffi (1.17.0-x64-mingw-ucrt)
+    ffi (1.17.0-x86_64-darwin)
+    ffi (1.17.0-x86_64-linux-gnu)
```

Comment:
Reasons for updating the Gemfile.lock file?

Reply:
I just did bundle install on my windows machine and it automatically installed the updated one.

Reply:
do you want me to revert that? @Ashparshp

Reply:
Yeah! please, if you can!

Reply:
done! kindly check @Ashparshp